client.go

```diff
@@ -4,16 +4,26 @@ import (
 	csvFile "github.com/cloudquery/filetypes/v2/csv"
 	jsonFile "github.com/cloudquery/filetypes/v2/json"
 	"github.com/cloudquery/filetypes/v2/parquet"
+	"github.com/cloudquery/filetypes/v2/types"
 )
 
 type Client struct {
 	spec *FileSpec
 
+	types.FileType
```

Comment:
This is used for all interactions with the underlying writer. For read tests, separate Clients are used which are still below.